common/src/test/java/tachyon/security/PlainSaslServerProviderTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security;
+
+import java.util.HashMap;
+
+import javax.security.sasl.Sasl;
+import javax.security.sasl.SaslServer;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import tachyon.security.PlainSaslHelper;
+
+public class PlainSaslServerProviderTest {
+
+  @Test
+  public void createPlainSaslServerTest() throws Exception {
+    // create plainSaslServer
+    SaslServer server = Sasl.createSaslServer("PLAIN", "", "",
+        new HashMap<String, String>(), null);
+    Assert.assertEquals("PLAIN", server.getMechanismName());
+  }
```

Comment:
Also adding a test case to pass some other string (e.g., "UNKNOWN_MACHANISM") to the factory method and check if the returned value is `null`?